test/system/accountant_workflow_test.rb

```diff
@@ -56,10 +55,37 @@ class AccountantWorkflowTest < ApplicationSystemTestCase
 
   describe 'viewing pledged patients' do
     it 'should show only pledged patients on page load' do
-      # Or initials, as the case may be
-      assert has_content? @fulfilled_patient.initials
-      assert has_content? @pledged_patient.initials
-      refute has_content? @nonpledged_patient.initials
+      assert has_content? @fulfilled_patient.name
+      assert has_content? @pledged_patient.name
+      refute has_content? @nonpledged_patient.name
+    end
+
+    it 'should properly display counts' do
+      assert has_content? 'Displaying all 3 patients'
+    end
+
+    it 'should properly display large numbers of patients' do
+      30.times do |i|
+        Patient.create! name: "Patient #{i}",
+                        appointment_date: 3.days.from_now,
+                        clinic: @clinic,
+                        fund_pledge: 100,
+                        initial_call_date: 3.days.ago,
+                        line: 'DC',
+                        naf_pledge: 200,
+                        patient_contribution: 100,
+                        pledge_sent: true,
+                        primary_phone: "999-888-#{1000 + i}",
+                        procedure_cost: 400
+      end
+
+      # gotta reload!
+      visit accountants_path
+
+      assert has_content? 'Displaying patients 1 - 25 of 33 in total'
+
+      # the table should have 25 rows
+      assert has_selector?('tbody#accountants-table-content tr', count: 25)
```

Comment:
Nit: I think the `tbody` makes this a little more confusing - would this work with just the ID and the `tr`?

Reply:
duh yup it's an id, no need to further clarify!